### ISSUE TITLE
install script doesnt check all files for new versions

### ISSUE THREAD

**Issue by saschadd** [mitchellkrogza/nginx-ultimate-bad-bot-blocker#87]
Hi mitchell,
me again with another issue.
I installed the blocker a while ago manually.
Now i noticed that there are some files changed since then and thought i will give the installer a try to update all files in one go.
There i noticed that the installer script checks bots.d folder and if it exists it checks if all files are there but not if the files on the github repo are newer or different.
This could be wanted because there are some files that can be changed but maybe an info note should state that there are newer files?!

**Comment by mitchellkrogza** [comment]
Hi @saschadd thanks, yes indeed this has crossed my mind already. I first need to start writing version information blocks into all those include files and then once that is done I am sure @itoffshore can modify update-ngxblocker to check for those newer versions and offer the user an option to download them (potentially over writing any customizations) or not download them. I will work on introducing the version info block in the coming week.

**Comment by mitchellkrogza** [comment]
Commit: https://github.com/mitchellkrogza/nginx-ultimate-bad-bot-blocker/commit/4e0af5ba0f3fc66ea5464156222b769ecab2bb16 introduces versioning information blocks to all include files.

**Comment by saschadd** [comment]
just tried but somehow it is not working as intended.
I did
cd /usr/sbin
rm update-ngxblocker
wget https://raw.githubusercontent.com/mitchellkrogza/nginx-ultimate-bad-bot-blocker/master/update-ngxblocker
chmod +x /usr/sbin/update-ngxblocker
./update-ngxblocker

and get

Latest Blacklist Already Installed: 3.2017.09.869

Emailing report to: me@myemail.com

Then i tried 
./update-ngxblocker -e name.lastname@subdomain.domain.tld
and get
grep: end of given interval not valid _(translated from german output)_
ERROR: option '-e' argument 'name.lastname@subdomain.domain.tld' requires: email@domain.com

The files in my bots.d folder are from 30. August, so they should be seen as outdated i think.

**Comment by itoffshore** [comment]
that fixes the email issue with busybox & GNU `grep`

**Comment by saschadd** [comment]
Email works now! Thank you itoffshore

**Comment by mitchellkrogza** [comment]
Excellent @saschadd glad @itoffshore got this solved quickly. I'm sure when he has some spare time he can improve the update script for checking the include files for new versions. Enjoy and report back on how it's working for you. I've seen such massive improvements on all my sites across the board since I started developing this a year ago.

**Comment by mitchellkrogza** [comment]
@itoffshore doe this now check for version information in include files? Sorry on my mobile so not able to really dig through Travis logs

**Comment by itoffshore** [comment]
`install-ngxblocker` does not check the include versions yet.

when I have time I'll modify it to download updated includes to `include_name.conf.new` & print a message.

**Comment by mitchellkrogza** [comment]
:+1:

**Comment by saschadd** [comment]
with curl it is possible to check files against their date.
maybe this could be a different approach for the not so often changing files which could make things easier when mitchell has not to update all files with version dates?!

**Comment by mitchellkrogza** [comment]
Thanks @saschadd I prefer the version information blocks, those files seldom every update so manually adding the incremented version number and Date to the version info block is much easier for me and for users to keep control over.
